src/lib/components/data-graphic/guides/Axis.svelte

```diff
@@ -173,12 +174,18 @@ This component will draw an axis on the specified side.
     // use graphicWidth or graphicHeight
     // do we ensure different spacing in one case vs. another?
     tickCount = ~~(axisLength / 20);
-    tickCount = Math.max(2, ~~(tickCount / 100));
+    const maxTickCountThreshold = 2 + ~~($plotConfig.width / 250);
+    tickCount = Math.max(maxTickCountThreshold, ~~(tickCount / 100));
```

Comment:
Something tells me that we should probably also have a smarter way of determining when to change the tick count. X axes will have overlap very easily and Y axes won't. I think we could change the tick count logic depending on these, which largely looks like what we divide this number by.

In the case of x, I think `tickCount / 100` seems pretty good. For y, I think it could probably be smaller. Could you try a few things and see what produces a useful / uncluttered number of ticks for the y axis? `tickCount / 50`  might work well (a tick every 50px, basically).

Reply:
Tried different settings and looks like 50 works well for `y`.  100 for `x` is getting too cluttered so have used 150